debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -371,7 +373,7 @@ private boolean schemaChanged(List<PgProto.DatumMessage> messageList, Table tabl
         // go through the list of columns from the message to figure out if any of them are new or have changed their type based
         // on what we have in the table metadata....
         return messageList.stream().filter(message -> {
-            String columnName = message.getColumnName();
+            final String columnName = PostgresTableIdTransformer.INSTANCE.fromSqlQuoted(message.getColumnName());
```

Comment:
I have the same question here as above. Should the transformer be stored within the `Column` and then applied accordingly. There could be a `NoOpTransformer` for unquoted column and table names. That way we'd also avoid the frequent `beginsWith()` calls.